app/src/main/res/layout/item_reservation.xml

```diff
@@ -1,13 +1,19 @@
 <?xml version="1.0" encoding="utf-8"?>
 <androidx.cardview.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
-    xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="182dp"
-    android:layout_marginStart="23dp"
-    android:layout_marginTop="25dp"
-    app:cardCornerRadius="26dp"
-    android:layout_marginEnd="23dp">
+    android:layout_marginStart="2dp"
+    android:layout_marginTop="15dp"
+    android:layout_marginEnd="2dp"
+    android:background="@drawable/reservation"
+
+    android:elevation="5dp"
+
+
+
+    android:outlineAmbientShadowColor="@color/cardview_shadow_end_color"
```

Comment:
In the XML file, this line is warned, bcs of its use in the API 28 of Android, your minimum API is 21 in your project. 
If you use this in an emulator with less or equal of API 21, this screen will not consider that attribute. Do the test to realised it.

Reply:
perfect i have configured my xml code style in the ide